DESCRIPTION

```diff
@@ -40,7 +40,8 @@ Imports:
     RColorBrewer,
     heatmaply,
     gganimate,
-    plotly
+    plotly,
+    grid
```

Comment:
There's a merge conflict here that's keeping CI from running.

Reply:
I'm not sure what happened here. I just added an import package. Do you have any idea?

Reply:
Yeah - I changed this file a few PRs back to add a different package. We're getting a merge conflict because your repo doesn't have my change and git thinks we are both trying to add to the end of the Imports list. 

Easiest thing to do is to manually sync your `develop` branch to the `develop` branch of this repo and then rebase your PR on that. See https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork for details on how to do the first part.

Reply:
I don't think that merge went properly.... it added a bunch of unrelated changes (undoing a lot of recent work) and still didn't fix this file...